src/modules/service-catalog/components/service-catalog-list-item/EmptyState.tsx

```diff
@@ -0,0 +1,63 @@
+import { Anchor, Button } from "@zendeskgarden/react-buttons";
+import { LG, MD } from "@zendeskgarden/react-typography";
+import styled from "styled-components";
+import { useTranslation } from "react-i18next";
+import { getColorV8 } from "@zendeskgarden/react-theming";
+
+const Container = styled.div`
+  padding: ${(p) => p.theme.space.xl} 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: ${(props) => props.theme.space.md};
+`;
+
+const TextContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  gap: ${(props) => props.theme.space.xxs};
+`;
+
+const StyledLink = styled.a`
+  &:visited,
+  &:hover {
+    text-decoration: none;
+    color: ${(props) => getColorV8("grey", 100, props.theme)};
+  }
+`;
+
+export const EmptyState = () => {
+  const { t } = useTranslation();
+  return (
+    <Container>
+      <TextContainer>
+        <LG>
```

Comment:
I would set it as an H2 using `tag="h2"`

Reply:
Is it needed for example for A11y? It adds bottom padding.

Reply:
I think it would be better for a11y but it is not strictly needed